### ISSUE TITLE
Security: stop panicking due to NU5 `unimplemented!()` statements

### ISSUE THREAD

**Issue by teor2345** [ZcashFoundation/zebra#2369]
## Motivation

When we were adding NU5 support to Zebra, we added comment TODOs and `unimplemented!()` placeholders to Zebra.

We need to make sure all the `unimplemented!()` placeholders have been removed. We should also check for any comment TODOs.

This ticket needs to be completed before NU5 testnet activation, otherwise Zebra will panic as soon as NU5 activates.

## Solution

Fixing panics:
- [ ] Turn `unimplemented!()` statements that haven't been implemented into tickets
- [ ] Remove `unimplemented!()` statements that have been implemented elsewhere

Task tracking (lower priority):
- [ ] Check for `NU5` or `ZIP` TODO comments, and make sure they have tickets

**Comment by teor2345** [comment]
@mpguerra this ticket needs to be completed before NU5 testnet activation, otherwise Zebra will panic as soon as NU5 activates.
